cloud-service-provider/ibm/deployable-architectures/productivity-suite/variables.tf

```diff
@@ -0,0 +1,271 @@
+# IBM Cloud Provider Variables
+variable "ibmcloud_api_key" {
+  description = "IBM Cloud API key"
+  type        = string
+}
+
+variable "email" {
+  description = "IBM cloud email address"
+  type        = string
+  default     = ""
+}
+
+# Kubeconfig file variable
+variable "kubeconfig_file" {
+  description = "Path to the kubeconfig file for the target Kubernetes cluster."
+  type        = string
+  default     = "~/.kube/config"
+}
+
+# API Tokens
+variable "hf_token" {
+  description = "HuggingFace API token for model access"
+  type        = string
+  sensitive   = true
+}
+
+variable "openai_api_key" {
+  description = "OpenAI API key for LLM services"
+  type        = string
+  sensitive   = true
+}
+
+# ChatQNA Variables
+variable "chatqna_model_dir" {
+  description = "Directory path for ChatQNA models"
+  type        = string
+}
+
+variable "chatqna_helm_repo" {
+  description = "Path to the ChatQNA Helm chart"
```

Comment:
"ChatQNA Helm Chart repo", add default value: ghcr.io/opea-project/charts/chatqna:1.3.0

Or even better: add one more variable opea_release and use that for for OPEA helm chart references.